src/components/Modals/Send/hooks/useFormSend/useFormSend.tsx

```diff
@@ -11,176 +11,199 @@ import {
   type UtxoChainId,
   utxoChainIds,
 } from '@shapeshiftoss/chain-adapters'
+import type { HDWallet } from '@shapeshiftoss/hdwallet-core'
 import { supportsETH } from '@shapeshiftoss/hdwallet-core'
 import type { KnownChainIds } from '@shapeshiftoss/types'
+import { useCallback } from 'react'
 import { useTranslate } from 'react-polyglot'
-import { useSelector } from 'react-redux'
 import { getChainAdapterManager } from 'context/PluginProvider/chainAdapterSingleton'
-import { useEvm } from 'hooks/useEvm/useEvm'
+import { getSupportedEvmChainIds } from 'hooks/useEvm/useEvm'
 import { useModal } from 'hooks/useModal/useModal'
 import { useWallet } from 'hooks/useWallet/useWallet'
 import { bn, bnOrZero } from 'lib/bignumber/bignumber'
 import { logger } from 'lib/logger'
 import { tokenOrUndefined } from 'lib/utils'
-import { selectPortfolioAccountMetadata } from 'state/slices/selectors'
+import { selectPortfolioAccountMetadataByAccountId } from 'state/slices/selectors'
+import { store } from 'state/store'
 
 import type { SendInput } from '../../Form'
 
 const moduleLogger = logger.child({ namespace: ['Modals', 'Send', 'Hooks', 'UseFormSend'] })
 
-export const useFormSend = () => {
-  const toast = useToast()
-  const translate = useTranslate()
+const handleSend = async ({
+  sendInput,
+  wallet,
+}: {
+  sendInput: SendInput
+  wallet: HDWallet
+}): Promise<string> => {
```

Comment:
Given the flow change without intermediary view-layer side-effects, we need to return the Txid to be consumed in `handleFormSend` (or any consumer that might need it)